contrib/miniupnpd/files/miniupnpd

```diff
@@ -0,0 +1,6 @@
+# miniupnpd service
+
+type = process
+command = /usr/bin/miniupnpd -f /etc/miniupnpd/miniupnpd.conf -d
```

Comment:
you can also pass --disable-fork to configure

aside from that, i think some other configure options might be useful?

Reply:
good catch, I missed `--disable-fork`.  Any particular configure option you believe we should enable?

Reply:
--vendorcfg allows editing mfg info in the config file manually later

maybe regex/pcp-peer/igd2? but i'm not sure how useful they are

Reply:
I never tried them as well, let me take a look

Reply:
I've checked alpine, arch and ubuntu and after that decided to enable vendorcfg, igd2, pvp-peer and leasefile.  I didn't see regex being used by anyone